mantidproject/mantid
# Always override MANTIDPATH inside workbench startup code

**Description of work.**

Disregards any `MANTIDPATH` that a user may have set when starting the new workbench. See the comment at the start of `mainwindow.py:main` for a more detailed explanation.

**Report to:** [nobody]. 

**To test:**

This needs to be tested on Linux.

Before merging the fix:
* Install an official build of mantid (apt/yum install mantid)
* log out and log back in (or source /etc/profile.d/mantid.sh)
* `echo $MANTIDPATH` should point to `/opt/Mantid/bin`
* build the workbench
* run `bin/workbench`
* -> it should fail to start.

Merge the fix and `bin/workbench` should successfully start the workbench.

*There is no associated issue.*

*This does not require release notes* because **the workbench is not advertised yet.**

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
